### PR TITLE
Create base structure for session page

### PR DESCRIPTION
**Pull request by harisumaid**

### CODE REVIEW

**Comment by harisumaid** [comment]
#20 
image

**Comment by aquibbaig** [comment]
Looks good!

**Comment by ganeshpatro321** [comment]
Please check the compatibility in different browsers. And, if possible share screenshots @harisumaid

**Comment by harisumaid** [comment]
FireFox
image

File: src/pages/sessions.js [hunk 1]
```diff
@@ -1,11 +1,157 @@
...
+import IndexContainer from "../containers/indexContainer";
 import NavigationBar from "../components/Navigationbar";
 import Footer from "../components/Footer";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
```
Comment (ganeshpatro321) [inline src/pages/sessions.js]: Remove Unnecessary imports.

File: src/pages/sessions.js [hunk 2]
```diff
@@ -1,11 +1,157 @@
...
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Divider from "@material-ui/core/Divider";
+import data1 from "../config/sessionToBeConducted.yaml";
```
Comment (ganeshpatro321) [inline src/pages/sessions.js]: Give better import names, will avoid confusion in the future. Else add comments describing what is data1.

Comment (ankitjena) [reply src/pages/sessions.js]: @harisumaid name them as the filenames. Always name variables such that another person knows the intent of the variable in one look

File: src/pages/sessions.js [hunk 3]
```diff
@@ -1,11 +1,157 @@
...
+    </div>
+  );
+};
+
+const SessionsAlreadyConducted = () => {
+  return (
+    <div style={title}>
+      <h1>Session already conducted</h1>
+    </div>
+  );
```
Comment (ankitjena) [inline src/pages/sessions.js]: The file is too long. Abstract this as two different  components. Keeps your components small. Three files altogether, one folder.

File: src/pages/sessions.js (hunk 1 above)
Comment (harisumaid) [reply src/pages/sessions.js]:
```suggestion
import React from "react";
import IndexContainer from "../containers/indexContainer";
import NavigationBar from "../components/Navigationbar";
import Footer from "../components/Footer";
import PropTypes from "prop-types";
import Card from "@material-ui/core/Card";
import CardActions from "@material-ui/core/CardActions";
import CardContent from "@material-ui/core/CardContent";
import Button from "@material-ui/core/Button";
import Grid from "@material-ui/core/Grid";
import Typography from "@material-ui/core/Typography";
import Divider from "@material-ui/core/Divider";
```
keeping these imports

File: src/pages/sessions.js (hunk 2 above)
Comment (harisumaid) [reply src/pages/sessions.js]:
```suggestion
import data1 from "../config/sessionToBeConducted.yaml";
// data1 contains data about the sessions that are not yet conducted
import data2 from "../config/sessionAlreadyConducted.yaml";
// data2 contains data about the sessions that we have already conducted

```
added those comments

Comment (ankitjena) [reply src/pages/sessions.js]: I would prefer renaming them.

Comment (ganeshpatro321) [reply src/pages/sessions.js]: Yes, rename them. Keep the names same as file names. That will be better.

File: src/pages/sessions.js (hunk 3 above)
Comment (harisumaid) [reply src/pages/sessions.js]:
```suggestion
  import SessionToBeConducted from "../components/session/sessionToBeConducted.js"
import SessionAlreadyConducted from "../components/session/sessionAlreadyConducted.js"

<div>
    <NavigationBar />
    <SessionToBeConducted/>
    <Divider style={grid} />
    <SessionAlreadyConducted/>
    <Footer />
  </div>
```
Abstracted those as different components in component folder

Comment (ankitjena) [reply src/pages/sessions.js]: Commit then!

**Comment by agathver** [comment]
Please make sure that the wording of commit messages, as well as PR titles and descriptions, are in a proper format.

Presentation is as important as code.

Read: https://chris.beams.io/posts/git-commit/

Think of a PR as a summary of all the work you did with your commits.

**Pull request merged**